pages/auth_zero.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
Please remove these two lines, this file is not meant to be run directly so this is not needed.